headers.asm

```diff
@@ -332,17 +331,17 @@ nt_bracket_char:
 
 nt_char_plus:
         .byte 5, 0
-        .word nt_chars, xt_one_plus, z_one_plus ; same as 1+
+        .word nt_chars, xt_char_plus, z_char_plus ; same as 1+
```

Comment:
i used the duplicated symbols which otherwise get flagged as undefined, and used the comment to note it's shared code.